### PR TITLE
Optimized Images in Meshery Page

### PR DESCRIPTION
**Pull request by deveshXm**
**Description**

This PR fixes #3135 

Optimized Images in Meshery Page

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d3a5e709c6bda8f51e5caa4b434d6a6a1c45b78b at: https://6310c81d4776d906557f1bbc--layer5.netlify.app

**Review by sahilsuman933 — APPROVED** [review]

File: src/sections/Meshery/Features-section/index.js [hunk 1]
```diff
@@ -62,15 +65,15 @@ const FeaturesSection = () => {
               </p>
             </Col>
             <Col xs={12} xl={8} className="smp-section-caraousel">
-              <Slider {...settings}>
+              <Slider {...settings} height={600}>
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Features-section/index.js:68]: This is causing issues in the mobile view of the slider. Why do we need this?

File: src/sections/Meshery/Meshery-mange-mesh/index.js [hunk 2]
```diff
@@ -46,7 +44,7 @@ const MesheryManageMesh = () => {
             </Button>
           </Col>
           <Col sm={12} lg={6}>
-            <StaticImage src={MultiMesh} alt="Meshery multi-mesh" className="multi-mesh-img"/>
+            <img src={MultiMesh} alt="Meshery multi-mesh" className="multi-mesh-img"/>
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Meshery-mange-mesh/index.js:47]: Revert this change. Why is this done?

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: src/sections/Meshery/Meshery-mange-mesh/index.js (hunk 2 above)
Comment (deveshXm) [reply src/sections/Meshery/Meshery-mange-mesh/index.js:47]: If StaticImage is used on .svg it gets blurred. If you will notice the image is blurred out but after changing it to img the image's quality got better.

Comment (Nikhil-Ladha) [reply src/sections/Meshery/Meshery-mange-mesh/index.js:47]: Ok, SVG!
Cool

**Comment by deveshXm** [comment]
@Nikhil-Ladha i did not change img1 and img2 in meshery-features to StaticImages as they were having css issues for mobile view. I tried to find the styles but i couldn't.

File: src/sections/Meshery/Features-section/features-section.style.js [hunk 3]
```diff
@@ -101,6 +101,11 @@ const FeaturesSectionWrapper = styled.section`
 							max-height: 20rem;
 							margin: auto;
 						}
+
+						StaticImage {
+							max-height: 20rem;
+							margin: auto;
+						}
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Features-section/features-section.style.js:108]: I don't think this will actually be applied, because the rendered element is simply an `img` wrapped within a `div`. The `StaticImage` component is just the naming done for it in the gatsby-image plugin.

File: src/sections/Meshery/Features-section/index.js [hunk 4]
```diff
@@ -67,10 +71,10 @@ const FeaturesSection = () => {
                   <img src={slide1} alt="Slide 1" />
                 </div>
                 <div>
-                  <img src={slide2} alt="Slide 2" />
+                  <StaticImage src={slide_2} alt="Slide 2" />
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Features-section/index.js:74]: Either use `StaticImage` for all, or for neither of them.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit e321c6bc70ea32e9f00aabfb82f096d3b4725fe4 at: https://631afef8eb3e3211168ab174--layer5.netlify.app

File: src/sections/Meshery/Features-section/features-section.style.js (hunk 3 above)
Comment (deveshXm) [reply src/sections/Meshery/Features-section/features-section.style.js:108]: Actually it worked for me. It did fix the styling for me.

File: src/sections/Meshery/Features-section/index.js (hunk 4 above)
Comment (deveshXm) [reply src/sections/Meshery/Features-section/index.js:74]: One is gif so i can't use static Images on it.

File: src/sections/Meshery/Features-section/features-section.style.js (hunk 3 above)
Comment (Nikhil-Ladha) [reply src/sections/Meshery/Features-section/features-section.style.js:108]: Yeah, just checked it seems like the CSS for that name is passed down to the `img` tags. Nice work from gatsby.
Though, it would be better to use `.gatsby-image-wrapper img` to assign CSS to images using the `StaticImage` component.

File: src/sections/Meshery/Features-section/index.js (hunk 4 above)
Comment (Nikhil-Ladha) [reply src/sections/Meshery/Features-section/index.js:74]: But that's not fixed, right?
The values for the slide images are passed as props, so they can be an image too.

Though, does it not work if you use `StaticImage` for a gif? Technically, the component should just bypass the processing and render it as it is.

File: src/sections/Meshery/Features-section/features-section.style.js (hunk 3 above)
Comment (deveshXm) [reply src/sections/Meshery/Features-section/features-section.style.js:108]: Okay 😄

File: src/sections/Meshery/Features-section/index.js (hunk 4 above)
Comment (deveshXm) [reply src/sections/Meshery/Features-section/index.js:74]: Yes, but when i used StaticImage for GIF it did not show anything.

Comment (Nikhil-Ladha) [reply src/sections/Meshery/Features-section/index.js:74]: Ack.

File: src/sections/Meshery/Features-section/features-section.style.js (hunk 3 above)
Comment (leecalcote) [reply src/sections/Meshery/Features-section/features-section.style.js:108]: Where did we land on this suggestion?

**Comment by leecalcote** [comment]
@deveshXm not hearing from you by end of this week, we'll assume that this PR is abandoned and close it. Thanks much for your efforts here. Do let us know if you're still working on this item or not, if you would, pretty please.

**Comment by deveshXm** [comment]
@leecalcote really sorry for this. Due to unfortunate circumstances  i will not be able to continue on this. I'd definitely love to contribute on meshery again sooner or later.
